docs/styles/transitions.md

```diff
@@ -20,11 +20,49 @@ and fade out when it is closed:
 }
 ```
 
-In order for the close transition to take effect, you will also need to pass
-the `closeTimeoutMS={150}` prop to each of your modals.
 
 The above example will apply the fade transition globally, affecting all modals
 whose `afterOpen` and `beforeClose` classes have not been set via the
 `className` prop.  To apply the transition to one modal only, you can change
 the above class names and pass an object to your modal's `className` prop as
 described in the [previous section](classes.md).
+
+In order for the close transition to take effect, you will also need to pass
+the `closeTimeoutMS={150}` prop to each of your modals.
```

Comment:
Maybe this can be rephrased, something like: `The close animation is controlled via timer which you can define using `closeTimeoutMS={number}`, where the value is the time interval in milliseconds. The value must be synchronized with the css transition you have specified.

What do you think?

Reply:
Ah, also this part:

The value must be synchronized with the css transition or style you have specified (because styles can also be used in react-modal).